frontend/src/components/Univers/style.jsx

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+
+export default styled.div`
+  color: white;
+  .main_container {
```

Comment:
snake_case, camelCase, kebab-case : I saw these three cases in your classes' names.
Chose one and fix it everywhere please